### DISCUSSION TITLE
[M-Z] (2024/25) Foglio 3 - Esercizio 11

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/algebra#502] [Esercizi M-Z] [answered]
![es.11](https://github.com/user-attachments/assets/80dcbbe7-336e-491c-a179-03406e89fc81)

**Comment by marcoseverini** [comment, marked as answer]
![immagine](https://github.com/user-attachments/assets/55a1ec96-fa26-4b95-8d80-a96780c62b2f)
![immagine](https://github.com/user-attachments/assets/9f13d55e-fa2e-46cc-9a9b-9acdf139f285)
fedele alla soluzione del professore
